src/Turbo/assets/src/turbo_stream_controller.ts

```diff
@@ -50,7 +52,7 @@ export default class extends Controller {
 
     connect() {
         if (this.url) {
-            this.es = new EventSource(this.url);
+            this.es = new EventSource(this.url, this.eventSourceOptionsValue);
```

Comment:
EventSource does not seems to accept "options" but only `{ withCredentials: bool }` or do i read this wrong ? 

* https://developer.mozilla.org/en-US/docs/Web/API/EventSource/EventSource
* https://html.spec.whatwg.org/multipage/server-sent-events.html#eventsource